native/test/src/tests/room/user_settings.rs

```diff
@@ -0,0 +1,138 @@
+use acter::UserRoomSettings;
```

Comment:
also added rust e2e tests for the new usersettings objects and all its fields and their updates